packages/pieces/openai/src/index.ts

```diff
@@ -14,7 +15,32 @@ It is strongly recommended that you add your credit card information to your Ope
 export const openaiAuth = PieceAuth.SecretText({
   description: markdownDescription,
   displayName: 'Api Key',
-  required: true
+  required: true,
+  validate: async (auth) => {
+    try{
+      console.log("-=-=-=-=-=-=-=-");
+      console.log(auth);
+      console.log("-=-=-=-=-=-=-=-");
+      await httpClient.sendRequest<{
+        data: { id: string }[];
+      }>({
+        url: 'https://api.openai.com/v1/models',
+        method: HttpMethod.GET,
+        authentication: {
+          type: AuthenticationType.BEARER_TOKEN,
+          token: auth.auth as string
+        }
+      });
+      return{
+        valid: true,
+      }
+    }catch(e){
+      return{
+        valid: false,
+        error: 'Invalid API token'
```

Comment:
I would try to use ask instead of model, just to make sure we avoid issues like 429 too and the error i would make it has more information like at least status code or message so they can debug further.